pkg/operator/encryption/migration_controller.go

```diff
@@ -198,46 +196,73 @@ func (c *migrationController) migrateKeysIfNeededAndRevisionStable() (resetProgr
 			continue
 		}
 
-		// storage migration takes a long time so we expose that via a distinct status change
-		if err := c.setProgressing(strings.Title(groupToHumanReadable(gr))+strings.Title(gr.Resource), "migrating resource %s.%s to new write key", groupToHumanReadable(gr), gr.Resource); err != nil {
-			return false, err
-		}
-
 		if needsMigration(writeSecret, gr) {
-			migrationErr := c.runStorageMigration(gr)
-			errs = append(errs, migrationErr)
-			if migrationErr != nil {
-				continue
+			// storage migration takes a long time so we expose that via a distinct status change
+			if err := c.setProgressing(strings.Title(groupToHumanReadable(gr))+strings.Title(gr.Resource), "migrating resource %s.%s to new write key", groupToHumanReadable(gr), gr.Resource); err != nil {
```

Comment:
shouldn't this be the other way around? `migrating resource %s.%s to new write key` as a first arg since it contains formating info.

Reply:
the first is the reason, second the message (with printf semantics)